tests/infra/ccf.py

```diff
@@ -274,40 +276,36 @@ def create_node(self, local_node_id, host, debug=False, perf=False):
     def remove_last_node(self):
         last_node = self.nodes.pop()
 
-    def add_node(self, new_node_info):
-        with self.find_primary()[0].member_client(format="json") as member_client:
-            j_result = member_client.rpc("add_node", new_node_info)
-
-        return j_result
-
-    def create_and_add_node(
-        self, lib_name, args, should_succeed=True, local_node_id=None
-    ):
+    def create_and_add_node(self, lib_name, args):
         forwarded_args = {
             arg: getattr(args, arg) for arg in infra.ccf.Network.node_args_to_forward
         }
-        if local_node_id is None:
-            local_node_id = self.get_next_local_node_id()
+        local_node_id = self.get_next_local_node_id()
         node_status = args.node_status or "pending"
         new_node = self.create_node(local_node_id, "localhost")
+
+        primary, _ = self.find_primary()
         new_node.start(
+            start_type=infra.remote.StartType.join,
```

Comment:
Should StartType be renamed to something else so that we don't have StartType.start and StartType.join? Something like CreationType or CreationMode? (not necessarily relevant to this PR)

Reply:
That could also be hidden behind start() and join(), each setting the right enum value with an awkwardly long name.

Reply:
I also meant in general in ccf code, not only on the python side

Reply:
As discussed offline, I've changed `StartType.start` to `StartType.new` (in both our python infra and CCF).